push-api/JSONParser/README.md

```diff
@@ -1,14 +1,28 @@
-### General usage:
+# Crawl and Push
+
+Scanning a folder and push every item to the Push-API
+
+### Usage
 
 The main file is JSONParser.py
+It will execute a 4 step process:
 
-You will need to add the PyYAML lib
+1: read the config file (yamlParser.py)
+2: scan a folder and create a JSON (Reader.py) (optional if docs.json is already populated)
+3: parse the JSON file (Parser.py)
+4: loop through the array in the JSON file and push (Pusher.py) info to the Push-API
 
+### Requirement
+
+Python 3.*
```

Comment:
Give a download link? :)

Maybe an unordered list would be nice for the requirements too.